### ISSUE TITLE
double click event when zoom drag

### ISSUE THREAD

**Issue by mpushki** [naver/billboard.js#2104]
## Description
I need to add processing by clicking on a line or dot
1. Is it possible to add a handler to whole line
2. Event calls twice when zoom = drag

```
bb.generate({
  data: {
    xs: {
      data1: "x1",
      data2: "x2"
    },
    columns: [
	    ["x1", 1, 2, 3, 4, 5],
            ["x2", 1, 2, 4, 5, 6],
	    ["data1", 4, 1, 6, 8, 10],
            ["data2", 5, 2, 6, 7, 8]
    ],
    type: "line", // for ESM specify as: line()
    onclick: (d) => {
    	console.log('hi')
    },
  },
  zoom: {
    	enabled: true,
    	type: "drag",
    },
  bindto: "#multipleXYLineChart"
});
```

## Steps to check or reproduce
https://jsfiddle.net/mpushki/7a1xqm9w/15/

**Comment by watnab** [comment]
https://github.com/naver/billboard.js/blob/9801594b0306449798ba14f1484db6ee26718249/src/ChartInternal/interactions/eventrect.ts#L493
https://github.com/naver/billboard.js/blob/9801594b0306449798ba14f1484db6ee26718249/src/ChartInternal/interactions/zoom.ts#L346

**Comment by watnab** [comment]
```javascript
    onclick: (d) => {
    	console.log(['hi', this.event.type])
    },
```
may log "mouseup" or "click".

**Comment by watnab** [comment]
Event calls once when `xs` is omitted.

**Comment by netil** [comment]
> 1) Is it possible to add a handler to whole line

you can bind event directly to the svg element by:
```js
  onafterinit: function() {
	this.$.line.lines.each(function() {
		// remove inline pointer-events to receive event
		this.parentNode.parentNode.removeAttribute("style");

		this.addEventListener("click", e => {
			console.log(e.target, "clicked the line");
		});
	});
```
but will notice clicking line element is little bit harder, because the line shape is very thiner.


> 2) Event calls twice when zoom = drag

It happens when `zoom.drag` option is used. 
The click event listener was set to fix #583, but it seems working fine removing it from the zoom drag flow.
I'll do the fix.
